fabfile.py

```diff
@@ -63,7 +63,7 @@ def up():
     compose_files = env.compose_files
     env.compose_files = [file for file in compose_files if file != 'docker-compose.worker.yml']
 
-    docker_compose('up -d')
+    docker_compose('up --remove-orphans -d')
```

Comment:
@pyrech Is it compatible with your system of worker? 
On line 64, the worker file is not loaded, so it means it will remove all workers each time `fab up` is executed or not?

Reply:
Nice spot, it was removed for that purpose.

https://github.com/jolicode/docker-starter/pull/48/files#diff-65218180447789ae9b6a89838c1966c9L62-R68